mypy/checkmember.py

```diff
@@ -616,3 +619,40 @@ def erase_to_bound(t: Type):
         if isinstance(t.item, TypeVarType):
             return TypeType(t.item.upper_bound)
     return t
+
+
+def find_type_from_bases(e: NameExpr):
```

Comment:
Is this the right file for this function? It's only called from checker.py now, and it doesn't seem to use anything else defined here.

Also the way you retrieve the containing class scares me a bit. I'd somehow expect the connection between the simple lvalue and the class in which it occurs to be more direct. How does other code figure out the class in which it occurs? Here you use essentially e.node.info.